src/mono/wasm/runtime/startup.ts

```diff
@@ -318,32 +323,28 @@ function mono_wasm_pre_init_essential(isWorker: boolean): void {
         Module.removeRunDependency("mono_wasm_pre_init_essential");
 }
 
-// runs in both blazor and non-blazor
```

Comment:
Was this comment incrorrect?

Reply:
There is no special more for blazor anymore, but it's unrelated to this PR